doc.go

```diff
@@ -21,15 +21,15 @@ It does this by providing stable, idiomatic interfaces for common uses like
 storage and databases. Think `database/sql` for cloud products.
 
 At the core of the Go CDK are common "portable types" implemented by cloud
-providers. For example, objects of the blob.Bucket portable type can be created
-using gcsblob.OpenBucket, s3blob.OpenBucket, or any other provider. Then, the
-blob.Bucket can be used throughout your application without worrying about
-the underlying implementation.
+services. For example, objects of the blob.Bucket portable type can be created
```

Comment:
"portable types". These are service-independent abstractions for interacting with any supported cloud services that have a corresponding driver implementation.
OR
"portable types" implemented for supported cloud services by service-dependent drivers.

Reply:
Reworded a bit, PTAL.